cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -67,7 +67,7 @@ case class InputParameterCommandPart(commandInputParameter: CommandInputParamete
         case cip: CommandInputParameter =>
           val localizedId = LocalName(FullyQualifiedName(cip.id).id)
           inputsMap.get(localizedId) match {
-            case Some(x) =>x
+            case Some(x) => valueMapper(x)
```

Comment:
ToL (a little outside scope) a `fold` on the option could let you specify `valueMapper` w/o arguments and generally reduce code here.

Reply:
I'll fix that in my next PR